### PR TITLE
Add Bintray documentation to README and installation doc

### PR DESCRIPTION
**Pull request by cdrage**
Adds information on nightly builds to the README and installation
document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,30 @@ source <(kompose completion bash)
...
+
+### Downloading the latest (master) binary
+
+"Nightly" binaries are generated on a per-merge basis and uploaded to [Bintray](https://bintray.com/kompose)
+
+__Linux and macOS:__
+
+```sh
+# Linux 
+curl -L https://kompose.bintray.com/kompose/latest/:kompose-linux-amd64 -o kompose
```
Comment (kadel) [inline README.md]: the link is not working :-( It returns file with  just `The requested path was not found.`

This link should work  https://dl.bintray.com/kedgeproject/kedge/latest/kedge-linux-amd64

Comment (kadel) [inline README.md]: the link is not working :-( It returns file with  just `The requested path was not found.`

This link should work  https://dl.bintray.com/kedgeproject/kedge/latest/kedge-linux-amd64

File: README.md [hunk 2]
```diff
@@ -69,7 +69,30 @@ source <(kompose completion bash)
...
+
+# macOS
+curl -L https://kompose.bintray.com/kompose/latest/:kompose-darwin-amd64 -o kompose
```
Comment (kadel) [inline README.md]: https://dl.bintray.com/kompose/kompose/latest/kompose-darwin-amd64

File: README.md [hunk 3]
```diff
@@ -69,7 +69,30 @@ source <(kompose completion bash)
...
+
+chmod +x kompose
+sudo mv ./kompose /usr/local/bin/kompose
+```
+
+__Windows:__
+
+Download from [Bintray](https://kompose.bintray.com/kompose/latest/:kompose-windows-amd64.exe) and add the binary to your PATH.
```
Comment (kadel) [inline README.md]: https://dl.bintray.com/kompose/kompose/latest/kompose-windows-amd64.exe

File: docs/installation.md [hunk 4]
```diff
@@ -23,6 +23,29 @@ __Windows:__
...
+
+#### Nightly / master release
+
+"Nightly" binaries are generated on a per-merge basis and uploaded to [Bintray](https://bintray.com/kompose)
+
+__Linux and macOS:__
+
+```sh
+# Linux 
+curl -L https://kompose.bintray.com/kompose/latest/:kompose-linux-amd64 -o kompose
```
Comment (kadel) [inline docs/installation.md]: https://dl.bintray.com/kedgeproject/kedge/latest/kedge-linux-amd64

File: docs/installation.md [hunk 5]
```diff
@@ -23,6 +23,29 @@ __Windows:__
...
+
+# macOS
+curl -L https://kompose.bintray.com/kompose/latest/:kompose-darwin-amd64 -o kompose
```
Comment (kadel) [inline docs/installation.md]: https://dl.bintray.com/kompose/kompose/latest/kompose-darwin-amd64

File: docs/installation.md [hunk 6]
```diff
@@ -23,6 +23,29 @@ __Windows:__
...
+
+chmod +x kompose
+sudo mv ./kompose /usr/local/bin/kompose
+```
+
+__Windows:__
+
+Download from [Bintray](https://kompose.bintray.com/kompose/latest/:kompose-windows-amd64.exe) and add the binary to your PATH.
```
Comment (kadel) [inline docs/installation.md]: https://dl.bintray.com/kompose/kompose/latest/kompose-windows-amd64.exe

**Comment by cdrage** [comment]
@kadel I used the links directly from bintray but they were wrong.. thanks for the review! Updated the PR.

File: README.md [hunk 7]
```diff
@@ -69,7 +69,30 @@ source <(kompose completion bash)
...
+
+### Downloading the latest (master) binary
+
+"Nightly" binaries are generated on a per-merge basis and uploaded to [Bintray](https://bintray.com/kompose)
+
+__Linux and macOS:__
+
+```sh
+# Linux 
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kompose-linux-amd64 -o kompose
```
Comment (surajnarwade) [inline README.md]: link is not working :(

`https://dl.bintray.com/kedgeproject/kedge/latest/kompose-linux-amd64` looks weird

File: README.md [hunk 8]
```diff
@@ -69,7 +69,30 @@ source <(kompose completion bash)
...
+
+# macOS
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kompose-darwin-amd64 -o kompose
```
Comment (surajnarwade) [inline README.md]: same here

File: README.md [hunk 9]
```diff
@@ -69,7 +69,30 @@ source <(kompose completion bash)
...
+
+chmod +x kompose
+sudo mv ./kompose /usr/local/bin/kompose
+```
+
+__Windows:__
+
+Download from [Bintray](https://dl.bintray.com/kedgeproject/kedge/latest/kompose-windows-amd64.exe) and add the binary to your PATH.
```
Comment (surajnarwade) [inline README.md]: error 404 for this link

File: docs/installation.md [hunk 10]
```diff
@@ -23,6 +23,29 @@ __Windows:__
...
+
+#### Nightly / master release
+
+"Nightly" binaries are generated on a per-merge basis and uploaded to [Bintray](https://bintray.com/kompose)
+
+__Linux and macOS:__
+
+```sh
+# Linux 
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kompose-linux-amd64 -o kompose
```
Comment (surajnarwade) [inline docs/installation.md]: same as above

File: docs/installation.md [hunk 11]
```diff
@@ -23,6 +23,29 @@ __Windows:__
...
+
+# macOS
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kompose-darwin-amd64 -o kompose
```
Comment (surajnarwade) [inline docs/installation.md]: same as above

File: docs/installation.md [hunk 12]
```diff
@@ -23,6 +23,29 @@ __Windows:__
...
+
+chmod +x kompose
+sudo mv ./kompose /usr/local/bin/kompose
+```
+
+__Windows:__
+
+Download from [Bintray](https://dl.bintray.com/kedgeproject/kedge/latest/kompose-windows-amd64.exe) and add the binary to your PATH.
```
Comment (surajnarwade) [inline docs/installation.md]: same as above

**Comment by surajnarwade** [comment]
If I goto this link `https://dl.bintray.com/kedgeproject/kedge/latest/`,
I can see,

* kedge-darwin-amd64
* kedge-linux-amd64
* kedge-linux-arm
* kedge-windows-amd64.exe

`kompose` is not there

File: README.md (hunk 7 above)
Comment (kadel) [inline README.md]: oh, my bad :-D 

`kedgeproject/kedge` part is obviously wrong (my copy/paste error) :-(

`kompose/kompose` is supposed be there.



correct link is: https://dl.bintray.com/kompose/kompose/latest/kompose-linux-amd64

**Comment by kadel** [comment]
you can find all the correct links here: https://dl.bintray.com/kompose/kompose/latest/

Sorry about confusion with Kedge :-(

**Comment by cdrage** [comment]
I think this it the longest review for a doc every from mistypes, haha. Both @kadel and I had brainfarts.

Updated (again!).

**Pull request merged**